sklearn/datasets/_svmlight_format.pyx

```diff
@@ -64,7 +64,10 @@ def _load_svmlight_file(f, dtype, bint multilabel, bint zero_based,
 
         target, features = line_parts[0], line_parts[1:]
```

Comment:
Separate to this issue, but should we also be handling the case that there are no active features?